### PR TITLE
Pipe patch

### PR DESCRIPTION
**Pull request by animeshsingh**
Quick patch for pipeline sample + main README changed to reflect the website.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/409)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,12 @@
...
 ### Learn More
-* Join our [Working Group](https://groups.google.com/forum/#!forum/kfserving) for meeting invites and discussion.
-* [Read the Docs](/docs).
-* [Examples](./docs/samples).
-* [API Docs](./docs/apis/README.md)
-* [Roadmap](/ROADMAP.md).
-* [KFServing 101 Slides](https://drive.google.com/file/d/16oqz6dhY5BR0u74pi9mDThU97Np__AFb/view).
-* [KFServing 101 Tech Talk](https://www.youtube.com/watch?v=hGIvlFADMhU).
-* This project is an evolution of the [original proposal in the Kubeflow repo](https://github.com/kubeflow/kubeflow/issues/2306). 
+To learn more about KFServing, how to deploy it as part of Kubeflow, how to use various supported features, and how to participate in the KFServing community, please follow the [KFServing docs on the KubefloW Website](https://www.kubeflow.org/docs/components/serving/kfserving/).
```
Comment (yuzisun) [inline README.md]: /kubefloW/kubeflow/

File: docs/samples/pipelines/sample-tf-pipeline.py [hunk 2]
```diff
@@ -27,10 +26,14 @@ def kfservingPipeline(
     action = 'create',
     model_name='tensorflow-sample',
     default_model_uri='gs://kfserving-samples/models/tensorflow/flowers',
-    canary_model_uri='gs://kfserving-samples/models/tensorflow/flowers-2',
+    canary_model_uri='gs://kfserving-samples/models/tensorflow/flowers',
```
Comment (yuzisun) [inline docs/samples/pipelines/sample-tf-pipeline.py]: should be `flowers-2`?

File: docs/samples/pipelines/sample-tf-pipeline.py [hunk 3]
```diff
@@ -27,10 +26,14 @@ def kfservingPipeline(
...
     canary_model_traffic_percentage='10',
     namespace='kubeflow',
-    framework='tensorflow'
+    framework='tensorflow',
+    default_custom_model_spec='{}',
+    canary_custom_model_spec='{}',
+    autoscaling_target=0,
+    kfserving_deployer_api=''
```
Comment (yuzisun) [inline docs/samples/pipelines/sample-tf-pipeline.py]: what's `kfserving_deployer_api` ?

Comment (animeshsingh) [reply docs/samples/pipelines/sample-tf-pipeline.py]: changed it kfserving_endpoint. This is for scenarios when kfserving is running remote.

File: docs/samples/pipelines/sample-tf-pipeline.py (hunk 2 above)
Comment (animeshsingh) [reply docs/samples/pipelines/sample-tf-pipeline.py]: Done

File: README.md (hunk 1 above)
Comment (animeshsingh) [reply README.md]: Done

**Pull request merged**